pkg/onchain/website/handler.go

```diff
@@ -52,13 +52,29 @@ func RedirectToDefaultResourceInterceptor(req *interceptor.Interceptor) *interce
 		return nil
 	}
 
-	prefixes := []string{"/browse/", "/thyra/"}
+	redirectPaths := []string{"/", "/home", "/search", "/websiteUploader", "/store"}
+
+	for _, path := range redirectPaths {
+		if req.Request.URL.Path == path || req.Request.URL.Path == path+"/" {
+			http.Redirect(
+				req.Writer,
+				req.Request,
+				path+"/index.html",
+				http.StatusSeeOther,
+			)
+
+			return nil
+		}
+	}
 
+	prefixes := []string{"/browse/", "/thyra/"}
```

Comment:
`/thyra` is still here ?

Reply:
I didn't touch that yet, I have to see if this is still needed

Reply:
@SlnPons do we still have the need for browse an address without having a local dns?

Reply:
@gregLibert 
I tried with <http://station.massa/browse/AS18YcPZ4gocjx1SHkT3wBdb61WPD68sj2yupGT8QGbnQpQEkWpf>, it redirect me to <http://station.massa/browse/AS18YcPZ4gocjx1SHkT3wBdb61WPD68sj2yupGT8QGbnQpQEkWpf/index.html>, it loead the index.html file, this file contains `<script src="/static/js/main.bc678dac.js"></script>` so it can't work because <http://station.massa/static/js/main.bc678dac.js> is not found. Maybe we need to clarify our web-on-chain rules? (do not request for URL starting with "/")

but fortunately it works well for flappy <http://station.massa/browse/AS1MExWLGdJEPK9hXEZN5bsLFUXeVowJiTNx9mWyFkSZud1Z89Em/index.html>

Reply:
@Thykof let's remove it as discussed